app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -29,9 +33,34 @@ const Entry = ({
 }): JSX.Element => {
   const [currentWeek, setCurrentWeek] = useState<IGameWeek['week']>(1);
   const [entries, setEntries] = useState<IEntry[]>([]);
+  const [leagueName, setLeagueName] = useState<string>('');
   const [loadingData, setLoadingData] = useState<boolean>(true);
+  const [survivors, setSurvivors] = useState<number>(0);
+  const [totalPlayers, setTotalPlayers] = useState<number>(0);
   const { user } = useDataStore((state) => state);
 
+  /**
+   * Fetches the current league name.
+   * @returns {Promise<void>}
+   */
+  useEffect(() => {
+    /**
+     * Fetches the current league name.
+     */
```

Comment:
These comments seem redundant, could you please remove/update one of these?

Reply:
Yup, good call. I copy+pasted my code and didn't remove the redundant one.

Reply:
@choir27 updated in latest commit.